application/controllers/SurveyAdministrationController.php

```diff
@@ -2360,7 +2360,11 @@ public function actionCopy()
         }
 
         $this->aData = $aData;
-        $this->render('importSurvey_view', $this->aData);
+        if ((App()->getConfig("editorEnabled")) && isset(($aImportResults['newsid']))) {
```

Comment:
we also need to check for the surveytheme being "fruity_twentythree" (soon we will have all themes able to open the new editor, but not yet)

Reply:
@twilligls applied the change, but unfortunately this will only work for fruitytwentythree, but will not work for themes inherited from it

Reply:
that's fine for now